index.Rmd

```diff
@@ -490,7 +491,7 @@ Results for adult ZINB models are shown below:
 tab_model(adult_ANA_reg,adult_Bmic_reg,adult_Bmiya_reg,title = "Adult Regression Results",dv.labels = c("A. phagocytophilum","B. microti","B. miyamotoi"),show.r2 = F,digits=4)
 ```
 
-Results for nymphal ZINB models are shown below:
+Results for nymphal ZINB models are shown below: 
```

Comment:
In this table, there were numbers that are super long and made the table weirdly shaped. If there's a way to put 22967927079040806912.0000 and 8982440234598033705589571099311183383749282907422061481998971819359758938786691463249920.0000 in scientific notation, that would be great.

Reply:
Yeah I realized this after. My new version should address this!